mrp_stock_owner_restriction/tests/test_mrp_stock_owner_restriction.py

```diff
@@ -0,0 +1,85 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from odoo.tests import Form
+from odoo.tests.common import TransactionCase
+
+
+class TestMrpStockOwnerRestriction(TransactionCase):
+    def setUp(self):
+        super(TestMrpStockOwnerRestriction, self).setUp()
+        self.company = self.env.ref("base.main_company")
+        self.manufacture_route = self.env.ref("mrp.route_warehouse0_manufacture")
+
+        self.product = self.env["product.product"].create(
+            {
+                "name": "test product",
+                "type": "product",
+            }
+        )
+        self.product_2 = self.env["product.product"].create(
+            {"name": "test component", "default_code": "PRD 2", "type": "product"}
+        )
```

Comment:
To make variables easier to understand.
```suggestion
        self.finished_product = self.env["product.product"].create(
            {"name": "test product", "type": "product"}
        )
        self.component = self.env["product.product"].create(
            {"name": "test component", "type": "product"}
        )
```